integration-tests/kafka/src/test/java/org/apache/camel/quarkus/component/kafka/it/CamelKafkaTest.java

```diff
@@ -86,4 +89,127 @@ void testQuarkusKafkaClientFactoryNotConfigured() {
                 .statusCode(200)
                 .body(is("true"));
     }
+
+    @Test
```

Comment:
Ok, I think I got it. My understanding is that the trick to start/stop foo/bar routes is just to have the message committed or not.
So, I wonder if it would be clearer with a single route that would call `manual.commitSync();` each 2 messages only ?
So the first message would be committed, the second not and so on....

Reply:
And the idea is to verify at some point if it's committed, or never ? I'm trying to understand how to replace the last startup

Reply:
Thinking aloud:
We would have a kind of counter in the process() statement
The process statement would do something like that setBody(counter)
Send "content" to kafka, assert seda as seen counter = 1 and counter 2 (so delivered uncommitted and then delivered committed).
That way, we wouldn't need the last startup ? Would it cover everything needed thought ?

Reply:
Yeah it could work. Let me  try this.
I'm not either fan of starting and stopping Routes.

Reply:
I pushed a fix, to have only one single Route.
I still need to stop and restart this one to redeliver the message

Reply:
I had the idea that it would work like jdbc, amq commits. But maybe in the context of kakfa it's something else. Maybe it's just an asynchronous commit that has the time to be executed with the %2 testing method. If any, does camel-kafka tests also stop/start routes ?

Reply:
Ok, then I would say let's do this also.